javascript/javascript1/week2/flight-booking/flight-booking.js

```diff
@@ -0,0 +1,20 @@
+function getFullname(firstname, surname, formalName = false, isMale = true) {
+  if (!firstname || !surname) {
+    return "!!!Both first name and last name must be provided";
+  }
+  if (!formalName) {
+    return firstname + " " + surname;
+  } else if (formalName === true && isMale === true) {
+    return "Lord" + " " + firstname + " " + surname;
+  } else if (formalName === true && isMale === false) {
```

Comment:
Can be written shorter as:

```suggestion
  } else if (!isMale) {
```